Tests/VersionTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * SortTest.php
+ *
+ * @category        Naneau
+ * @package         SemVer
+ */
+
+use Naneau\SemVer\Parser;
+
+use \PHPUnit_Framework_TestCase as TestCase;
+
+/**
+ * SortTest
```

Comment:
Docblock is incorrect (class name is wrong), same goes for `SortTest.php` above.